package.json

```diff
@@ -227,7 +227,7 @@
     "jest-environment-jsdom": "^29.3.1",
     "jest-localstorage-mock": "^2.4.22",
     "jest-watch-typeahead": "^2.2.1",
-    "jsonwebtoken": "^8.5.1",
+    "jsonwebtoken": "9.0.0",
```

Comment:
What happened to the carret `^`?

What's your process for upgrading libraries?
- Manually enter the version
- `yarn install`
?

Reply:
I prefer to fix the versions to specific ones. Here we should do the same as we upgrade packages once in a while